includes/view/Questions_view.php

```diff
@@ -9,32 +9,35 @@
 function Questions_view($open_questions, $answered_questions, $ask_action)
 {
     foreach ($open_questions as &$question) {
-        $question['actions'] = form([
+        $question->actions = form([
```

Comment:
Updating the function docblock to contain `* @param Question[] $open_questions` is really helpful when using an IDE

Reply:
> A merge is not a [rebase](https://help.github.com/en/github/using-git/about-git-rebase#rebasing-commits-against-a-branch) and i see some opportunities to squash some commits together to have a cleaner history ;)

I am fairly new to rebase as I usually work with branches at work.
My apologies for this.


> You should really use your IDE's auto formatting feature or use a linter for that as there are some whitespace problems in the code ;)

Yeah, I'm using phpstorm but I didn't set it up for the engelsystem. My bad, I will go over the changes again with a configured phpstorm. 

> Another benefit is that a proper IDE can use docblock hints to know which magic method is available for a class which is rally nice if working with models

I use phpstorm but I am strongly against the use of magic methods. I will implement them though to keep the project consistent and to reduce some complexity :) 


> Following are some comments on the code:

Thank you for taking the time to review, I will try to have the changes done by today

Reply:
PHPStorm has a nice auto formatter, setup should only be selecting PSR-2 (or 12).

Well as magic methods make it much more clearer they should definitely be preferred ;)